backend/api/src/reaction.ts

```diff
@@ -111,15 +111,7 @@ export const addOrRemoveReaction: APIHandler<'react'> = async (props, auth) => {
           .eq('comment_id', contentId)
         
         // Broadcast the updated comment
-        const updatedComment = await db
-          .from('contract_comments')
-          .select()
-          .eq('comment_id', contentId)
-          .single()
-        
-        if (updatedComment.data) {
-          broadcast(`comment:${contentId}`, { type: 'comment_update', data: updatedComment.data })
-        }
+        broadcastCommentUpdate(contentId, { comment_id: contentId, likes: count ?? 0 })
```

Comment:
You need another helper function to broadcast the new like. Broadcasting the updated comment is no longer the right approach, you should delete all the stuff about updating a comment.